R/tm_data_table.R

```diff
@@ -282,6 +299,7 @@ ui_data_table <- function(id,
   tagList(
     teal.widgets::get_dt_rows(ns("data_table"), ns("dt_rows")),
     fluidRow(
+      ui_teal_transform_data(ns("decorate"), transformators = decorators),
```

Comment:
@m7pr the UI had the wrong id and was in the `ui_page_data_table` which created a different namespace.

Moving it to the "dataname" tab fixes the problem and allows each one to have their own decorator.